.github/workflows/pre-commit.yaml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [main]

```

Comment:
Looks like the default branch is called `master` in this repo, not `main`

Reply:
Indeed. df884ab